frontend/src/components/ApplicantInformation/index.tsx

```diff
@@ -23,10 +24,21 @@ import GeneticClassesType from '../../types/GeneticClasses';
 import getUrl from '../../utils/ApiUtils';
 import ApiAddresses from '../../utils/ApiAddresses';
 import { useAuth } from '../../contexts/AuthContext';
+import { FilterObj, filterInput } from '../../utils/filterUtils';
 
 import './styles.scss';
 
+interface ComboBoxEvent {
+  selectedItem: any
```

Comment:
Maybe trying using a different type here instead of `any`, I guess it can be `string|null`.

Reply:
having null doesn't work, changed to `string`

Reply:
Great, thanks!